setup.py

```diff
@@ -56,6 +56,7 @@
         "websurfer": ["beautifulsoup4", "markdownify", "pdfminer.six", "pathvalidate"],
         "redis": ["redis"],
         "ipython": ["jupyter-client>=8.6.0", "ipykernel>=6.29.0"],
+        "local-jupyter-exec": ["jupyter-kernel-gateway", "websocket-client", "requests", "ipykernel"],
```

Comment:
The meaning and difference of these two dependencies are not obvious. Need renaming or documentation.

Reply:
`ipython` will be removed when the upstream bug in jupyter gateway is fixed and is only being kept around because that bug is a showstopper for windows on the new impl